tool/src/query-to-elm.ts

```diff
@@ -90,8 +99,14 @@ let moduleName = 'GraphQL.' + filename;
 
 let outPath = path.join(path.dirname(graphqlFile), filename + '.elm');
 
-let url = introspectionUrl + '?query=' + encodeURIComponent(introspectionQuery.replace(/\n/g, '')); 
-request(url, function (err, res, body) {
+let url = introspectionUrl;
+let data = {query: introspectionQuery.replace(/\n/g, '')};
+let method = options["method"] || "GET";
+let reqopts = method == "GET" ? {url:url, method:method, qs:data}
+                              : {url:url, method:method,
+                                 headers: [{"Conent-Type": "application/json"}],
```

Comment:
Hey! Like you, I'll need to query with POST request my GraphQL server (Apollo) and I'm planning to use `elm-graphql`. 

I found what you both did awesome guys! Learning a lot while looking at the code! 👍 

There is this typo on the headers, `Conent` -> `Content`. Maybe this is what breaks the command line with `--method`?